### DISCUSSION TITLE
Бриф

### DISCUSSION THREAD

**Discussion by Darolga** [torin-asakura/masterok#1] [General]
**О Компании**
1.	Расскажите о компании, ее структуре, о ключевых особенностях ведения бизнеса в вашей сфере.
2.	На чем зарабатывает ваш бизнес? 
3.	Какой ваш основной продукт?
4.	Есть ли маркетинговый план? На какой уровень хотите выйти и каких целей достичь?
5.	Какие проблемы стоят перед вашим бизнесом в стратегии развития?
6.	Какие сложности у компании? 
7.	Как хотите упростить жизнь себе и пользователям?

**Пользователь**
1.	Кто ваша целевая аудитория? Возраст, пол, социальное положение, интересы.
2.	Если это несколько сегментов (B2B или B2C сектор) опишите каждый и с какой целью приходит на сайт.
а)
б)
в)
3.	Опишите основные сценарии взаимодействия (Кто и как будет пользоваться сервисом?)
4.	Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса)
5.	Проводит ли ваша компания исследование пользователей? (Яндекс Метрика, Гугл Аналитикс) (Хорошо бы получить доступ)
6.	Какое количество новых клиентов хотите получать ежемесячно?
7.	Какое количество постоянных клиентов хотите поддерживать в активном режиме?
8.	Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?
9.	Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются (хотели бы выкладывать)
10.	Как хотите упростить жизнь себе и пользователям?

**Роль сайта**
1.	Уже есть интернет-магазин. Какие цели и задачи должен решить новый?
2.	Какие финансовые выгоды вы ожидаете получить?
3.	Для какой части целевой аудитории мы создаем сайт?
4.	На чем зарабатывает сайт? Каких финансовых результатов ожидаете достичь?
5.	Какие ключевые продукты сайта? Что пользуется спросом? 
6.	Какие продукты предполагается продвигать в будущем?
7.	Как сайт влияет/ должен повлиять/ на ваш основной вид деятельности?
8.	Опишите проблемы существующего сайта, которые требуют решения.

**Конкуренты**
1.	Кто ваши основные конкуренты? 
2.	Какие у вас схожие продукты?
3.	Ведут ли они деятельность, которую не ведете вы?
4.	На кого вы ориентируетесь как пример?
5.	Есть ли у конкурентов сайты и какую роль он выполняет?
6.	Покажите решения конкурентов, которые вам нравятся именно вам и почему?
7.	Какие преимущества будут у вашего сайта в сравнение с конкурентами?
8.	Чем вы лучше других? (Ваши отличительные преимущества, уникальные предложения.)
9.	Какие барьеры? Что мешает, тормозит развитие? 
10.	Как хотите масштабироваться?

**Структура и функционал**
1.	Перечислите и опишите основные разделы вашего сайта.
2.	Перечислите и опишите основные сервисы сайта.
3.	Оцените периодичность обновления информации на сайте.
4.	С каких устройств пользователи будут пользоваться сервисом?
5.	Откуда пользователи будут попадать на сервис?
6.	В каких регионах будет работать сервис?
7.	Кто помимо пользователей будет использовать систему? 
8.	Кто будет заниматься наполнением сайта контентом?
9.	На какой функционал и элементы нужно обратить особенное внимание?
10.	Есть ли функционал, который запланирован на будущее, но сейчас не будет реализован?  
11.	Будет ли внедрена программа лояльности?

**Дизайн и стилистика**
1.	Есть ли у вашей компании фирменный стиль, логотип, рекламные материалы? Требуется ли их разработка (или доработка имеющегося)? Каким должен быть?
2.	Перечислите несколько сайтов, которые вам нравятся, вне зависимости от тематики. Чем они вас привлекают?
3.	Перечислите несколько сайтов, которые вам не нравятся, вне зависимости от тематики. Чем они вас отталкивают?
4.	Есть ли примеры похожих проектов?
5.	Опишите ваш будущий сайт несколькими словами. Каким вы его видите?
6.	Есть ли узнаваемость бренда сейчас?
7.	Какие проблемы вашего бизнесом бренд должен помочь решить?

**Comment by EXPOMET** [comment]
1.Строительный рынок "МАСТЕРОК" занимается продажей строительных и отделочных материалов. Рынок состоит из конкурирующих Индивидуальных Предпринимателей занимающихся торговлей. Товар в основном есть в наличии на 75% и под заказ 25%.
2.Организация зарабатывает на перепродаже стройматериалов.
3.Основным продуктом являются: Электро товары, краски, сантехника.
4.Планируем заниматься продажей всех строительных и отделочных материалов в сети интернет.
5.Есть желание развития интернет-площадки.
Пользователь
1.10% - ООО, 20%- производители работ среднего возраста, 70%- городские посетители. 30%- женский пол, 70%- мужской пол. Возрастная категория 19-50 лет.
2. На сайт приходит категория В2С - с целью докупить не достающую продукцию, либо те у кого нет времени на посещение рынка.
3. Сервисом будут пользоваться менеджеры организаций с учётом оплаты товара по безналичному расчёту, а так же покупатели экономящие своё время.
4. В идеале: покупатель зайдя на сайт выбирает нужный товар , оплачивает его с доставкой.
5. Исследование пользователей не производилось.
6. Кол-во постоянных и приходящих клиентов - по максимуму.
7. Связь осуществляется с помощью мобильной связи, приложения WhatssApp. e-mail.
8. Соц. сети не ведутся. Хотели бы выкладывать новинки товаров, а так же скидки и понижения цен.
9.  Сайт должен быть по максимуму автоматизирован для быстрого взаимодествия с покупателями.
 Роль сайта
1.Интернет-магазин есть на интернет страничке рынка. Новый сайт должен привлечь пользовательскую аудиторию, действовать как торговая площадка, содержать максимально возможный ассортимент.
3. Аудитория: покупатели близ лежащей территории округа ( производители работ, частники), организации занимающиеся ремонтом и строительством.
4. Сайт зарабатывает на продажах и предоставлении места для торговли сторонним организациям.
5. ключевые товары электро товары, краски, сантехника.
6. Правильно работающий сайт приведёт к расширению ассортимента продукции на 70% и дальнейшему развитию организации с соответствующим приростом прибыли, а так же появится возможность для запуска новых проектов.
7. Проблема существующего сайта в том, что он изначально создавался как рекламный сайт строительного рынка без создания интернет-магазина. Плохая IT - поддержка сторонних организаций не заинтересованных в работе сайта привела его к неработоспособности. Требуется привести сайт в порядок, с частичной переделкой.
Конкуренты
1. "Петрович", рынок "Владимирский тракт", "Леруа Мерлен", рынок "Пехорка".
2. Продукция одинаковых марок, плюс у первых трёх перечисленных конкурентов большой выбор в наличии.
3. нет не ведут.
4. Ни на кого не ориентируемся, ведём среднюю ценовую политику.
5. Сайты конкурентов предлагают весь ассортимент + доставка.
8. Наше отличие от конкурентов - это быстрая доставка в течении получаса, "Петрович" - в течении дня или 2 часа.
9. Для развития требуется увеличение покупателей, что приведёт к наращиванию объёма продукции.  
10. Хочется постоянного роста и развития.
Структура и функционал
1. Основные разделы сайта: электрика, обои,крепёж.
2. Сервис: доставка, подъём.
4. Попасть на сайт можно зайти с мобильных устройств и компьютерной техники.
5. Приход пользователей будет зависеть от рекламы.
6. Регион работы приложения: г. Балашиха и округ на расстоянии примерно 30км.
7. Наполнением сайта и контентом будет заниматься менеджер.
8. Программа лояльности обязательна.
Дизайн и стилистика
1. Логотип имеется, но требует доработки.
3. Узнаваемость бренда есть.
